scripts/release/release.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/sh
+
+SCRIPTS_DIR=$(dirname "$0")
+BASE_DIR="$(pwd)"
+SRC_DIR="$BASE_DIR/libelektra"
+BUILD_DIR="$SRC_DIR/build"
+FTP_DIR="$BASE_DIR/ftp"
+DOC_DIR="$BASE_DIR/docu"
+
+install_elektra() {
+	echo "Installing elektra..."
+	rm -rf $BUILD_DIR && mkdir $BUILD_DIR && cd $BUILD_DIR && cmake -DBUILD_SHARED=ON -DBUILD_FULL=ON -DBUILD_STATIC=ON ..
+	cd ..
+	cmake --build build -- -j5
```

Comment:
maybe we can avoid hard-coding 5 here. At least let us put it in a variable.

Reply:
Afaik in our docker environment, if you run here `withDockerEnv` you have the number of cpus as a variable. @ingwinlu has defined some quite sane defaults there:
```
"MAKEFLAGS='-j${cpu_count+2} -l${cpu_count*2}'"
```

If `make` is run within this environment, you would actually automatically profit from the already set `MAKEFLAGS`, but you overrode those values with the command line args.

Please use something like this everywhere and do not manually set the parallelism level. I won't comment on the other instances of the same problem.